pyaedt/modeler/actors.py

```diff
@@ -348,14 +353,14 @@ def insert(self, app, motion=False):
 
         Parameters
         ----------
-        app: class: `pyaedt.hfss.Hfss`
+        app: pyaedt.Hfss
         motion: bool, optional
-            The default is ``False``.
+            Whether the actor is in motion. The default is ``False``.
 
         Returns
         -------
         list
-            List of antenna placed.
+            List of antenna that have been placed.
```

Comment:
```suggestion
            List of antennae that have been placed.
```
I believe it should be plural of antenna here.